torch/CMakeLists.txt

```diff
@@ -347,45 +337,76 @@ install(TARGETS torch
   LIBRARY DESTINATION "${TORCH_INSTALL_LIB_DIR}"
   ARCHIVE DESTINATION "${TORCH_INSTALL_LIB_DIR}")
 
-if (TORCH_BUILD_TEST)
-  # JIT Tests. TODO: Put into test/cpp/jit folder
-
+# JIT Tests. TODO: Put into test/cpp/jit folder
```

Comment:
Don't we still have a flag that toggles if tests build or not? (Or am I misremembering?)

Reply:
it used to default to ON and there were no consumers that set it OFF. I can reintroduce it if you like

Reply:
If we always build tests, then this is harmless, but I was under the impression that some people liked to do builds without having to build tests (I'm not one of them.)

Reply:
++ I'll re-add it then